modules/update-service-uninstall-web-console.adoc

```diff
@@ -5,14 +5,12 @@ You can use the {product-title} web console to uninstall the OpenShift Update Se
 
 .Prerequisites
 
-- All OpenShift Update Service applications have been deleted.
+* All OpenShift Update Service applications have been deleted.
 
 .Procedure
 
-To uninstall the OpenShift Update Service Operator using the {product-title} web console:
+. In the web console, click *Operators* -> *Installed Operators*.
 
-. In the {product-title} web console, click *Operators* -> *Installed Operators*.
-
-. Click *Uninstall Operator* from the menu of *OpenShift Update Service* from the list of installed Operators.
+. Select *OpenShift Update Service* from the list of installed Operators and click *Uninstall Operator*.
 
 . From the *Uninstall Operator?* confirmation dialog, click *Uninstall* to confirm the uninstall.
```

Comment:
Should uninstall be uninstallation?

Reply:
Yep! Looked it up and ISG confirms. Resolved.